src/styles.css

```diff
@@ -33,11 +37,13 @@
 
 .home {
   background-image: url("../media/images/header-illsutration-mobile@2x.png");
-  background-size: cover;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position-x: right ;
```

Comment:
An easier way to do back ground is shorthand, which would be "background-position: 100% 100%;" (or a different %)
That way you can move things around easier, and also test it easier with devtools in the browser.

Reply:
thanks it worked great